packages/eslint-plugin-shopify/lib/config/rules/babel.js

```diff
@@ -2,22 +2,12 @@
 // also turned off in configs that include these rules.
 
 module.exports = {
-  // Handles destructuring arrays with flow type in function parameters
-  'babel/array-bracket-spacing': ['warn', 'never'],
-  // Handles async functions correctly
-  'babel/arrow-parens': ['warn', 'always'],
-  // Require a particular separator between properties in Flow object types.
-  'babel/flow-object-type': ['warn', 'comma'],
-  // Require or forbid trailing commas for function paramater lists.
-  'babel/func-params-comma-dangle': ['warn', 'always-multiline'],
```

Comment:
Just for my own sanity, this rule is still applied, but is done by the default `comma-dangle` rule, yes?